org.mixedrealitytoolkit.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboardTouchAdapter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clauseusing System;
+
+using Microsoft.MixedReality.GraphicsTools;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace MixedReality.Toolkit.UX.Experimental
+{
+    public class NonNativeKeyboardTouchAdapter : MonoBehaviour
```

Comment:
All public, protected types and members need to be documented with a `<summary>` tag.   Please add a `<summary>` tag,

Reply:
Done